web/static/css/vendor-overrides/_trix.scss

```diff
@@ -0,0 +1,50 @@
+trix-editor {
+  @include border-top-radius(0);
+}
+
+trix-toolbar {
+  @include border-top-radius($base-border-radius);
+  background-color: $white;
+  border: $dark-border;
+  border-bottom-width: 0;
+
+  .button_groups {
+    font-size: 0;
+  }
+
+  .button_group {
+    border: 0;
+    border-radius: 0;
+    margin: 0;
+
+    button,
+    input[type=button] {
+      @include size(42px);
+      border: 0;
+      border-radius: 0;
+      color: $base-font-color;
+
+      &:not(:first-child) {
+        border: 0;
+      }
+    }
+  }
+
+  .strike,
+  .quote,
+  .code,
+  .block-level,
+  .history_tools {
```

Comment:
I kept this in the same order that they appear in the DOM. Agree though, alphabetizing probably makes more sense, especially if we're coming back to remove one.